fix(curriculum): worked on early feedbacks for js role playing game (#49169)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a3c2fccf186146b59c6e96.md

# --description--

`button1` represents your first `button` element. These elements have a special property called `onclick`, which you can use to determine what happens when someone clicks that button.

> Properties in JavaScript can be accessed in a couple of ways - the first is with <dfn>dot notation</dfn>. Accessing the `onclick` property of a button would look like 

Reviewer: ```suggestion
Properties in JavaScript can be accessed in a couple of ways - the first is with <dfn>dot notation</dfn>. Accessing the `onclick` property of a button would look like this:
```

> Properties in JavaScript can be accessed in a couple of ways - the first is with <dfn>dot notation</dfn>. Accessing the `onclick` property of a button would look like 

Reviewer: Tried to simplify this a bit and rewrite it to use active voice:

```suggestion
You can access properties in JavaScript a couple of different ways. The first is with <dfn>dot notation</dfn>. Accessing the `onclick` property of a button would look like this:
```

Author: I've made the change locally.

> You can access properties in JavaScript a couple of different ways. The first is with <dfn>dot notation</dfn>. Accessing the `onclick` property of a button would look like 

Reviewer 2: ```suggestion
You can access properties in JavaScript a couple of different ways. The first is with <dfn>dot notation</dfn>. Accessing the `onclick` property of a button would look like:
```
